src/Compilers/VisualBasic/Portable/Binding/Binder_Utils.vb

```diff
@@ -669,58 +686,58 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                 End If
 
                 ' default type is object.
-                Return GetSpecialType(SpecialType.System_Object, identifier, diagBag)
+                Return GetSpecialType(specialType.System_Object, identifier, diagBag)
```

Comment:
>specialType [](start = 38, length = 11)

This doesn't look like an intentional capitalization change. Please revert. Same for changes in the function below